<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IN</class>
 <widget class="QDialog" name="IN">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PARK</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/new/prefix1/Image/Mantravar.png</normaloff>:/new/prefix1/Image/Mantravar.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(194, 194, 194);</string>
   </property>
   <property name="text">
    <string>ADMIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="parklot">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>791</width>
     <height>521</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="parkSlot_1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="parkSlot_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="parkSlot_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="parkSlot_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>400</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="parkSlot_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>400</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="parkSlot_6">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>400</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="parkSlot_7">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>400</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="parkSlot_8">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>400</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="parkSlot_9">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>160</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="parkSlot_10">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>370</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="parkSlot_11">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="parkSlot_12">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="parkSlot_13">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="parkSlot_14">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>300</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="parkSlot_15">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>300</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Tempus Sans ITC&quot;;
text-decoration: underline;</string>
   </property>
   <property name="text">
    <string>PARKING ZONE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>9</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
